integrations/webchat/examples/custom-element/client/react-animation/src/config.js

```diff
@@ -6,6 +6,9 @@ const config = {
   region: 'us-south',
   serviceInstanceID: '9a3613d2-3ce6-4928-8eb6-4d659d87ae68',
   subscriptionID: null,
+  // You would likely use our own launcher in this scenario.
```

Comment:
If you want square corners like the JS example, you should add the corners option.